zyte_common_items/processors.py

```diff
@@ -100,8 +120,8 @@ def brand_processor(value: Union[Selector, HtmlElement], page: Any) -> Any:
     return extract_brand_name(value, search_depth=2)
 
 
-@only_handle_nodes
-def price_processor(value: Union[Selector, HtmlElement], page: Any) -> Any:
+@_handle_price
+def price_processor(value: Union[Selector, HtmlElement, str, None], page: Any) -> Any:
```

Comment:
@kmike do we want to pass the strings returned from the user-defined field to pass through or parse them?

Reply:
It seems it'd be good to have passthrough for a processor which is enabled by default. Currently all other processors do a passthrough for strings, so it seems it'd be good to follow this.

Reply:
So, when the input is JSON, should users [wrap strings with a `parsel.Selector`](https://github.com/zytedata/zyte-common-items/pull/69#discussion_r1432308724) to get the processor to work?

Reply:
It seems it'd be good to have passthrough for a processor which is enabled by default. Currently all other processors do a passthrough for strings, so it seems it'd be good to follow this.

Reply:
> So, when the input is JSON, should users https://github.com/zytedata/zyte-common-items/pull/69#discussion_r1432308724 to get the processor to work?

I think they can just call the price parsing function directly in this case.

Reply:
We can also provide more aggressive, non-transparent processors, not enabled by default. But I'm not sure we should do it right now.